### PR TITLE
Fixup readme

### PR DESCRIPTION
**Pull request by 2m**
and remove the duplication after #163.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -8,19 +8,10 @@ Paradox [![scaladex-badge][]][scaladex] [![travis-badge][]][travis] [![gitter-ba
 [gitter]:                    https://gitter.im/lightbend/paradox
 [gitter-badge]:       https://badges.gitter.im/lightbend/paradox.svg
 
-Paradox is a markdown documentation tool for software projects.
+Paradox is a markdown documentation tool for software projects. See [Paradox docs](http://developer.lightbend.com/docs/paradox/latest/) for details.
 
-**Paradox is NOT supported under the Lightbend subscription.**
-
-See [Paradox docs](http://developer.lightbend.com/docs/paradox/latest/) for details.
+**Paradox is NOT supported under the Lightbend subscription.** The project is maintained by the [Paradox Team](https://github.com/orgs/lightbend/teams/paradox) and all [the contributors](https://github.com/lightbend/paradox/graphs/contributors). Pull requests are very welcome–thanks in advance!
```
Comment (bantonsson) [inline README.markdown]: I actually like to have these on separate lines.

Comment (2m) [reply README.markdown]: Sure

**Comment by 2m** [comment]
No problem. Easy fix. :shipit:

**Comment by 2m** [comment]
Ahh, then I removed the wrong duplication. Was there another project you have based this off?

**Comment by ktoso** [comment]
Yeah, the style was based on https://github.com/lightbend/genjavadoc and we should add such thing to all projects under lightbend organisation. Here perhaps your wording is much better, but let's put it as the last section, under license - perhaps `# Maintanance notes`?

**Comment by 2m** [comment]
Maintenance notes it is. Pushed the changes. Check it out: https://github.com/lightbend/paradox/tree/wip-fix-readme-2m

**Comment by ktoso** [comment]
Not waiting for appveyor for readme change

**Pull request merged**